core/src/main/scala/pureconfig/BasicConverters.scala

```diff
@@ -90,15 +90,15 @@ trait TypesafeConfigConverters extends ConvertHelpers {
   implicit val configConfigConvert: ConfigConvert[Config] = new ConfigConvert[Config] {
     override def from(config: ConfigValue): Either[ConfigReaderFailures, Config] = config match {
       case co: ConfigObject => Right(co.toConfig)
-      case other => fail(WrongType(other.valueType().toString, "ConfigObject", ConfigValueLocation(config), None))
+      case other => fail(WrongType(s"${other.valueType}", s"${ConfigValueType.OBJECT}", ConfigValueLocation(config), None))
```

Comment:
I'm maybe too picky but when I prefer to use the standard `.toString` instead of string interpolation for one value. Same for all the other occurrences.